worldedit-core/src/main/java/com/sk89q/worldedit/extent/world/WorldApplyingExtent.java

```diff
@@ -27,64 +27,37 @@
 import com.sk89q.worldedit.function.operation.RunContext;
 import com.sk89q.worldedit.math.BlockVector2;
 import com.sk89q.worldedit.math.BlockVector3;
+import com.sk89q.worldedit.util.SideEffectApplier;
+import com.sk89q.worldedit.util.collection.BlockMap;
 import com.sk89q.worldedit.world.World;
+import com.sk89q.worldedit.world.block.BlockState;
 import com.sk89q.worldedit.world.block.BlockStateHolder;
-import com.sk89q.worldedit.world.block.BlockTypes;
 
 import java.util.HashSet;
 import java.util.Iterator;
-import java.util.List;
+import java.util.Map;
 import java.util.Set;
 
 /**
- * Implements "fast mode" which may skip physics, lighting, etc.
+ * An extent that sets blocks in the world, with BlockUpdateOptions.
  */
-public class FastModeExtent extends AbstractDelegateExtent {
+public class WorldApplyingExtent extends AbstractDelegateExtent {
```

Comment:
I don't know if I really like this name. Does `SideEffectExtent` sound good?

Reply:
Possibly, yeah - my main reason for naming it this, is it's the extent that actually writes the changes to the world